andela/ah-nyati-frontend
# #166840972 Implement social authentication (auth)

#### What does this PR do?
- Implements the feature that enables a user to register/login to the application using his social account (Google, Facebook, Twitter, or Github)

#### Description of Task to be completed?
- Create social auth component
- Write unit tests

#### How should this be manually tested?
- Follow the directions on the README to set up a local copy of the project
- In your terminal, navigate to the root directory of the project on your local machine and run
  `npm run dev` to start the development server
- Open **http://localhost:3000** on your browser. You should see the application homepage displayed
- Click on the _login_ link in the top navigation menu. You will be directed to the login page.
- Click on any of the social login buttons, enter your details if prompted. On successful verification, you should be directed to the dashboard page of the application

#### Any background context you want to provide?
- 

#### What are the relevant pivotal tracker stories?
- [#166840972](https://www.pivotaltracker.com/story/show/166840972)

#### Screenshots (if appropriate)
- 
<img width="1440" alt="Screenshot 2019-08-22 at 8 08 24 AM" src="https://user-images.githubusercontent.com/36275424/63493740-1ced9380-c4b4-11e9-877b-d8c2dc28412a.png">


#### Questions:
-
